lib/Tests/Fhp/ResponseTest/ResponseTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Fhp\ResponseTest;
+
+use Fhp\Response\Response;
+
+class ResponseTest extends \PHPUnit_Framework_TestCase
+{
+    protected static function getMethod( $class, $name )
+    {
+        $class  = new \ReflectionClass( $class );
+        $method = $class->getMethod( $name );
+        $method->setAccessible( TRUE );
+
+        return $method;
+    }
+
+    public function test_getter_and_setter()
+    {
+        $response = self::getMethod( 'Fhp\Response\Response', 'splitSegment' );
+
+        $withoutEscape = new Response( '' );
+        $escaped       = clone $withoutEscape;
+
+        $segments        = $response->invokeArgs( $withoutEscape, [
+            'HISAL:5:5:3+111111111::280:111111111+GiroBest+EUR+C:9999,99:EUR:20161018+C:0,:EUR:20161018+0,:EUR+9999,99:EUR',
+        ] );
+
+        $segmentsEscaped = $response->invokeArgs( $escaped, [
+            'HISAL:5:5:3+111111111::280:111111111+GiroBusiness?++EUR+C:9999,99:EUR:20161018+C:0,:EUR:20161018+0,:EUR+9999,99:EUR',
+        ] );
+
+        $this->assertEquals( 'HISAL:5:5:3', $segments[ 0 ] );
```

Comment:
Please change
`$this->assertEquals( 'HISAL:5:5:3', $segments[ 0 ] );`
to
`$this->assertEquals('HISAL:5:5:3', $segments[0]);`